### PR TITLE
Add Plume65

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds the Plume65 keyboard by Plume Keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plume/plume65/rules.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+UNICODE_ENABLE = no         # Unicode
```
Comment (fauxpark) [inline keyboards/plume/plume65/rules.mk:21]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/plume/plume65/readme.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Plume65
+
+## Support
+* Keyboard Maintainer: maartenwut
```
Comment (noroadsleft) [inline keyboards/plume/plume65/readme.md:4]:
```suggestion
* Keyboard Maintainer: [maartenwut](https://github.com/maartenwut)
```

File: keyboards/plume/plume65/plume65.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
+/* Copyright 2020 Dekkers
```
Comment (noroadsleft) [inline keyboards/plume/plume65/plume65.h:1]: The Split Backspace layouts also need their layout data added to `info.json`.

File: keyboards/plume/plume65/config.h [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+// #define SOFT_SERIAL_PIN D0 // or D1, D2, D3, E6
+
+// #define BACKLIGHT_PIN B7
+// #define BACKLIGHT_BREATHING
+// #define BACKLIGHT_LEVELS 3
+
```
Comment (noroadsleft) [inline keyboards/plume/plume65/config.h:57]:
```suggestion
```

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**